Yamllint doesn't show line numbers

## Describe the bug

Yamllint's output doesn't have line numbers. How it can be fixed?

```
2021-06-17 09:06:34 [ERROR]   Error code: 1. Command output:
------
Error: [quoted-strings] string value is not quoted with single quotes
Error: [quoted-strings] string value is not quoted with single quotes
Error: [quoted-strings] string value is not quoted with single quotes
Error: [quoted-strings] string value is not quoted with single quotes
Error: [quoted-strings] string value is not quoted with single quotes
Error: [quoted-strings] string value is not quoted with single quotes
Error: [quoted-strings] string value is not quoted with single quotes
Warning: [line-length] line too long (183 > 180 characters)
Warning: [comments] too few spaces before comment
Warning: [comments] too few spaces before comment
Warning: [comments] too few spaces before comment
Warning: [line-length] line too long (257 > 180 characters)
Warning: [line-length] line too long (335 > 180 characters)
Warning: [line-length] line too long (260 > 180 characters)
Warning: [line-length] line too long (259 > 180 characters)
Warning: [line-length] line too long (259 > 180 characters)
Warning: [line-length] line too long (260 > 180 characters)
Warning: [line-length] line too long (314 > 180 characters)
Warning: [comments] too few spaces before comment
Warning: [comments] too few spaces before comment`
```

## Config files

.yaml-lint.yml:
```
---

extends: 'default'

yaml-files:
  - '*.yml'
  - '*.yml.j2'

rules:
  line-length:
    max: 180
    level: 'warning'
  quoted-strings:
    quote-type: 'single'
```

linter config:
```
---
name: 'Lint action'

on:
  push:
    branches-ignore: ['master']
  pull_request:
    branches: ['master']

jobs:
  build:
    name: 'Code Base Lint'
    runs-on: 'ubuntu-latest'

    steps:
      - name: 'Checkout Code'
        uses: 'actions/checkout@v2'
        with:
          # Full git history is needed to get a proper list of changed files within `super-linter`
          fetch-depth: 0

      ################################
      # Run Linter against code base #
      ################################
      - name: 'Run Super Linter'
        uses: 'docker://ghcr.io/github/super-linter:slim-v4'
        env:
          ANSIBLE_CONFIG_FILE: '.ansible-lint.yml'
          ANSIBLE_DIRECTORY: 'ansible'
          DEFAULT_BRANCH: 'master'
          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
          LINTER_RULES_PATH: '/'
          PYTHON_FLAKE8_CONFIG_FILE: 'tox.ini'
          VALIDATE_ALL_CODEBASE: false
          VALIDATE_ANSIBLE: true
          VALIDATE_BASH: true
          VALIDATE_PYTHON_FLAKE8: true
          VALIDATE_PYTHON: true
          VALIDATE_YAML: true
          YAML_CONFIG_FILE: '.yaml-lint.yml'
```

--- a/lib/linter.sh
+++ b/lib/linter.sh
@@ -879,7 +879,7 @@ LINTER_COMMANDS_ARRAY['TSX']="eslint --no-eslintrc -c ${TSX_LINTER_RULES}"
 LINTER_COMMANDS_ARRAY['TYPESCRIPT_ES']="eslint --no-eslintrc -c ${TYPESCRIPT_ES_LINTER_RULES}"
 LINTER_COMMANDS_ARRAY['TYPESCRIPT_STANDARD']="standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin ${TYPESCRIPT_STANDARD_LINTER_RULES}"
 LINTER_COMMANDS_ARRAY['XML']="xmllint"
-LINTER_COMMANDS_ARRAY['YAML']="yamllint -c ${YAML_LINTER_RULES} -f standard"
+LINTER_COMMANDS_ARRAY['YAML']="yamllint -c ${YAML_LINTER_RULES} -f parsable"
 
 debug "--- Linter commands ---"
 debug "-----------------------"

--- a/lib/linter.sh
+++ b/lib/linter.sh
@@ -879,7 +879,7 @@ LINTER_COMMANDS_ARRAY['TSX']="eslint --no-eslintrc -c ${TSX_LINTER_RULES}"
 LINTER_COMMANDS_ARRAY['TYPESCRIPT_ES']="eslint --no-eslintrc -c ${TYPESCRIPT_ES_LINTER_RULES}"
 LINTER_COMMANDS_ARRAY['TYPESCRIPT_STANDARD']="standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin ${TYPESCRIPT_STANDARD_LINTER_RULES}"
 LINTER_COMMANDS_ARRAY['XML']="xmllint"
-LINTER_COMMANDS_ARRAY['YAML']="yamllint -c ${YAML_LINTER_RULES}"
+LINTER_COMMANDS_ARRAY['YAML']="yamllint -c ${YAML_LINTER_RULES} -f standard"
 
 debug "--- Linter commands ---"
 debug "-----------------------"

